client/extensions/wp-job-manager/components/setup/intro/index.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Steps } from '../constants';
+import Button from 'components/button';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import SectionHeader from 'components/section-header';
+
+class Intro extends Component {
+	skipSetup = () => this.props.goToStep( Steps.CONFIRMATION );
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Setup Wizard Introduction' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'WordPress setup wizard' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpjm-setup-wizard-intro)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).